odoo/tests/runner.py

```diff
@@ -102,6 +102,8 @@ def addFailure(self, test, err):
     def addSubTest(self, test, subtest, err):
         # since addSubTest is not making a call to addFailure or addError we need to manage it too
         # https://github.com/python/cpython/blob/3.7/Lib/unittest/result.py#L136
+        if self._soft_fail:
+            return
```

Comment:
Actually we should keep the logError since it can give a usefull piece of information, it should be lowered to an INFO by lower logging

And maybe just avoid to call super here